### DISCUSSION TITLE
what's the exact meaning of anchor?

### DISCUSSION THREAD

**Discussion by sparkok** [excaliburjs/Excalibur#2578] [Q&A] [answered]
I am new,and I can't understand the exact meaning of 'anchor'.who can tell me that.

**Comment by tenpaMk2** [comment, marked as answer]
Hi @sparkok  .
Indeed, the [doc](https://excaliburjs.com/docs/api/edge/classes/GraphicsComponent.html#anchor) is lacking in explanation, I think.

`anchor` is a normalized vector that determines where the graphic is based.
I made a diagram below. Do you understand it?

![excalibur-anchor](https://user-images.githubusercontent.com/8945311/219018629-0a5d9e58-92e2-4c88-9345-286b15283881.png)

(I haven't developed Excalibur.js for a while, so sorry if I'm wrong.)

**Reply by ewal** [reply]
> Hi @sparkok . Indeed, the [doc](https://excaliburjs.com/docs/api/edge/classes/GraphicsComponent.html#anchor) is lacking in explanation, I think.
> 
> `anchor` is a normalized vector that determines where the graphic is based. I made a diagram below. Do you understand it?
> 
> ![excalibur-anchor](https://user-images.githubusercontent.com/8945311/219018629-0a5d9e58-92e2-4c88-9345-286b15283881.png)
> 
> (I haven't developed Excalibur.js for a while, so sorry if I'm wrong.)

Nice explanation. Perhaps make a PR to the docs repo?

**Reply by eonarheim** [reply]
@tenpaMk2 This is a fantastic diagram! I would 100% support adding this to the docs repo!

**Reply by sparkok** [reply]
I got it,thank you!

**Reply by tenpaMk2** [reply]
> @tenpaMk2 This is a fantastic diagram! I would 100% support adding this to the docs repo!

Thanks @eonarheim ❗
Please tell me where the [docs](https://excaliburjs.com/docs/) repo (NOT [Edge API Doc](https://excaliburjs.com/docs/api/edge/) repo).

I will send a pull req.
I will do the belows.

1. Refine the image.
2. Add the image to [Graphics Component](https://excaliburjs.com/docs/graphics-component/#component-specific-overrides).
3. Link other pages to ↑.

If you have more good idea, please tell me.

**Reply by eonarheim** [reply]
@tenpaMk2 Awesome! I support your proposed changes!

The docs repo is here: https://github.com/excaliburjs/excaliburjs.github.io/tree/site/docs

Making changes to any of the markdown files in the `docs` directory will show up on the website https://excaliburjs.com/docs/

Some possible places to link/update as well to include information about anchors
* Actor Graphics  https://github.com/excaliburjs/excaliburjs.github.io/blob/site/docs/03-actors.mdx#actor-graphics
